connectors/src/connectors/confluence/temporal/workflows.ts

```diff
@@ -39,6 +40,8 @@ const {
   startToCloseTimeout: "20 minutes",
 });
 
+const MAX_HISTORY_LENGTH = 30_000;
```

Comment:
nit: TEMPORAL_WORKFLOW_MAX...
with com explaining for the future noob why this is there (a single confluence workflow may visit more than X pages, this breaks...)